android/src/main/java/com/onegini/mobile/sdk/reactnative/handlers/registration/RegistrationRequestHandler.kt

```diff
@@ -0,0 +1,36 @@
+//@todo Later will be transferred to RN Wrapper
```

Comment:
It's a good practice to add ticket links in comments like this. Deletion of those can be missed and other developer won't know if that was done or not.

Reply:
There were multiple of these comments which don't seem to be related to any existing tickets. I have removed them all as they don't seem to add any value.